backend/src/test.ts

```diff
@@ -0,0 +1,16 @@
+// scheduler.ts
+import Bree from "bree";
```

Comment:
Conceptually you don't need a 3rd party scheduler to run this on a 5 second cadence. If this is intended to be a queue processor you just need a module to run on start up. Once the module runs on start up it should read stuff from the DB, perform actions as needed and then go to sleep for whatever duration your business needs are.